types/block.go

```diff
@@ -676,9 +676,9 @@ func (cs CommitSig) ValidateBasic() error {
 		if len(cs.Signature) == 0 {
 			return errors.New("signature is missing")
 		}
-		if len(cs.Signature) > MaxSignatureSize {
-			return fmt.Errorf("signature is too big (max: %d)", MaxSignatureSize)
-		}
+		// if len(cs.Signature) > MaxSignatureSize {
```

Comment:
I commented this out when trying to make all `e2e` tests pass (when forward-porting uncompressed `secp256k1`. We may want to replace the commented if by another one that checks the signature size depending on its type (if it can be done)

Reply:
If may want to restore this "if" as well

Reply:
Done [63d4874](https://github.com/0xPolygon/cometbft/pull/2/commits/63d4874af86e62d77d85cb7013249ee552ded987)

Reply:
Resolved